makefile

```diff
@@ -50,6 +50,8 @@ docker: build/containerbuddy_build consul etcd
 build: docker
 	${DOCKERMAKE} build
 
+build/containerbuddy: build
```

Comment:
Neither this change nor the way it was seems to be working the way I'd expect with respect to rebuilds. Given that this is orthogonal to the point of this PR, I'm going to back this change out and fix it under separate issue.